app/models/step.rb

```diff
@@ -6,8 +6,10 @@ class Step < BaseResource
     string :source
   end
 
-  def get_status(steps_completed)
-    if (steps_completed.to_i + 1) == order
+  def get_status(steps_completed, status)
```

Comment:
Should there be constants for any of this stuff? `error` and `complete` show up in a couple of places.